docs/instrumentation/pymssql/pymssql.rst

```diff
@@ -0,0 +1,7 @@
+OpenTelemetry pymssql Instrumentation
+=====================================
+
+.. automodule:: opentelemetry.instrumentation.pymssql
```

Comment:
Pls add the relevant needed pymssql version to docs-requirements.txt - This can be verified using `tox -e docs`